src/server/command/worldedit/blueprint.zig

```diff
@@ -0,0 +1,170 @@
+const std = @import("std");
+
+const main = @import("root");
+const User = main.server.User;
+const vec = main.vec;
+const Vec3i = vec.Vec3i;
+
+const openDir = main.files.openDir;
+const List = main.List;
+const Block = main.blocks.Block;
+const Blueprint = main.blueprint.Blueprint;
+const NeverFailingAllocator = main.heap.NeverFailingAllocator;
+
+pub const description = "Input-output operations on blueprints.";
+pub const usage =
+	\\/blueprint save <file-name>
+	\\/blueprint delete <file-name>
+	\\/blueprint load <file-name>
+	\\/blueprint list
+;
+
+const BlueprintSubCommand = enum {
+	save,
+	delete,
+	load,
+	list,
+	other,
+	empty,
+
+	fn fromString(string: []const u8) BlueprintSubCommand {
+		return std.meta.stringToEnum(BlueprintSubCommand, string) orelse {
+			if(string.len == 0) return .empty;
+			return .other;
+		};
+	}
+};
+
+pub fn execute(args: []const u8, source: *User) void {
+	var argsList = List([]const u8).init(main.stackAllocator);
+	defer argsList.deinit();
+
+	var splitIterator = std.mem.splitScalar(u8, args, ' ');
+	while(splitIterator.next()) |a| {
+		argsList.append(a);
+	}
```

Comment:
Actually that this is not really the way the split iterator is intended to be used.
Ideally you'd avoid ever storing a list of arguments, and the function you call would just (attempt to) consumes the next item of the iterator.
Well, but now you've implemented it like this, and it's not really a huge deal so I won't force you to change this all again, it's probably not a big deal here anyways.
I just wanted to let you know what the intended usage is. Zig and its standard library are trying to guide you towards the simpler and faster choices, but it's not always quite clear what to do.